submissions/AbrahamJudah/PHP/index1.php

```diff
@@ -0,0 +1,61 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Products Log</title>
+    <link rel="stylesheet" type="text/css"  href="prod.css">
+</head>
+<body>
+    <div class= "container">
+        <h2>Product Logging Page</h2>
+        <form action="function1.php" method="POST">
+            <label>Product Name</label><br>
+            <input type="text" name="product"  required="required"><br>
+            <label>Quantity</label><br>
+            <input type="text" name="number"  required="required"><br>
+            <label>Date</label><br>
+            <input type="datetime-local" name="timestamp" required="required"><br>
+            <select name="brand" class="brand"  required="required">
+                <option value="id">Brand id</option>
+                <option value="1">Apple</option>
+                <option value="2">Armani</option>
+                <option value="3">Burberry</option>
```

Comment:
Good Job @Chigoz169 

I believe you can have this brand class in an array then fetch from the array instead of having a long list of option tags.